src/App.tsx

```diff
@@ -317,7 +314,8 @@ const App = (): JSX.Element => {
           await window.bridge.interBtcApi.tokens.subscribeToBalance(
             COLLATERAL_TOKEN,
             address,
-            (_, balance) => {
+            // TODO: should type properly
+            (_: any, balance: any) => {
```

Comment:
@danhper 
Please help me to type it correctly.

Reply:
Please have a look at how the library implements this and which type it returns. If you have questions around that please ask, but it's not Dan's task to look up types from the lib.

Reply:
@nud3l 
I agree with you. I guess I tend to ask questions that I should figure out myself.
@savudani8 
Please leave it to me. Sorry..

Reply:
FYI: I figured it out. I should have handled it myself from the beginning.